### PR TITLE
[3510] add rollover environment

### PR DESCRIPTION
**Pull request by misaka**
### Context

We're creating a rollover env to test rollover.

### Changes proposed in this pull request

Add a `rollover` Rails env.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by misaka** [comment]
[Add rollover RAILS_ENV](https://trello.com/c/bi6frhLQ/3510-add-rollover-railsenv)

File: README.md [hunk 1]
```diff
@@ -229,9 +229,9 @@ In order to use the external environment functionality you must set the
 environments in ```config/azure_environments.yml ```like so:  
 ```
 qa:
-    webapp: <webapp>
-    rgroup: <rgroup>
-    subscription: <subscription>
+  webapp: <webapp>
```
Comment (defong) [inline README.md:232]: is it missing rollover?

`mcb Command` do we need to add a bit concerning rollover for the future?

Comment (misaka) [reply README.md:232]: I think it's temporary and not worth adding, tbh.

Comment (defong) [reply README.md:232]: until next year then

**Pull request merged**